### ISSUE TITLE
Add support for modals

### ISSUE THREAD

**Issue by sam-moreton** [CalderaWP/cf-translate#18]
I'm running the plugin with the Avada wp theme and WPML, I tried adding a caldera form into a modal using [caldera_form_modal id=""] shortcode, the form displays in a modal and functions fine but it's not translated and the styling doesn't look right.

**Comment by Shelob9** [comment]
@freelancesam You're right, that's not going to work in a modal. I will try and get that fixed for the next release.

Need to:
- [ ] Initial form that is hidden is loaded correctly, based on current language.
- [ ] Refresh via language switcher targets the right element inside of the modal
- [ ] Works with multiple modals

**Comment by sam-moreton** [comment]
Thanks @Shelob9, Do you have an ETA for the next release? I need to know whether to remove modals for now or wait for the fix.

**Comment by Shelob9** [comment]
@freelancesam No, I don't have an ETA. If you need this right now, I would recommend using non-modal forms.

**Comment by sam-moreton** [comment]
Any news on the next release? could really do with translated modal forms
